### DISCUSSION TITLE
post/homelab-journey-future/

### DISCUSSION THREAD

**Discussion by giscus** [ecwu/ecwu.github.io#10] [Blog Post Discussion]
# post/homelab-journey-future/

Homepage and Blog of ECWU

https://ecwuuuuu.com/post/homelab-journey-future/

**Comment by TerenceLiu98** [comment]
催更 🫡🫡🫡 希望作者速速更新

**Comment by WallBreakerNO4** [comment]
不懂就问，为啥OP上要装emby😂

**Reply by ecwu** [reply]
复制粘贴忘了改 :P
